$MirOS$

	Simply regenerated, so that this port does not build-depend
	on (any!) autoconf port.

--- configure.orig	Sun Nov 22 00:02:01 2009
+++ configure	Sun Nov 22 00:02:08 2009
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.61 for GNU M4 1.4.9.
+# Generated by GNU Autoconf 2.61-MirPorts-1 for GNU M4 1.4.9.
 #
 # Report bugs to <bug-m4@gnu.org>.
 #
@@ -1376,8 +1376,7 @@ Optional Features:
 Optional Packages:
   --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
   --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
-  --without-included-regex
-                          don't compile regex; this is the default on 32-bit
+  --without-included-regexdon't compile regex; this is the default on 32-bit
                           systems with recent-enough versions of the GNU C
                           Library (use with caution on other systems). On
                           systems with 64-bit ptrdiff_t and 32-bit int,
@@ -1461,7 +1460,7 @@ test -n "$ac_init_help" && exit $ac_stat
 if $ac_init_version; then
   cat <<\_ACEOF
 GNU M4 configure 1.4.9
-generated by GNU Autoconf 2.61
+generated by GNU Autoconf 2.61-MirPorts-1
 
 Copyright (C) 1992, 1993, 1994, 1995, 1996, 1998, 1999, 2000, 2001,
 2002, 2003, 2004, 2005, 2006 Free Software Foundation, Inc.
@@ -1475,7 +1474,7 @@ This file contains any messages produced
 running configure, to aid debugging if configure makes a mistake.
 
 It was created by GNU M4 $as_me 1.4.9, which was
-generated by GNU Autoconf 2.61.  Invocation command line was
+generated by GNU Autoconf 2.61-MirPorts-1.  Invocation command line was
 
   $ $0 $@
 
@@ -1859,7 +1858,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 am__api_version='1.10'
 
 ac_aux_dir=
-for ac_dir in "$srcdir" "$srcdir/.." "$srcdir/../.."; do
+for ac_dir in $GNUSYSTEM_AUX_DIR "$srcdir" "$srcdir/.." "$srcdir/../.."; do
   if test -f "$ac_dir/install-sh"; then
     ac_aux_dir=$ac_dir
     ac_install_sh="$ac_aux_dir/install-sh -c"
@@ -1955,7 +1954,7 @@ fi
     # value for INSTALL within a source directory, because that will
     # break other packages using the cache if that directory is
     # removed, or if the value is a relative name.
-    INSTALL=$ac_install_sh
+    INSTALL="$SHELL $ac_install_sh"
   fi
 fi
 { echo "$as_me:$LINENO: result: $INSTALL" >&5
@@ -5802,7 +5801,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -5879,6 +5880,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -6313,7 +6315,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -6390,6 +6394,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
 
 cat >>confdefs.h <<_ACEOF
@@ -6848,7 +6853,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -6925,6 +6932,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -7297,7 +7305,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -7374,6 +7384,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -10149,7 +10160,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -10226,6 +10239,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -10261,7 +10275,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -10338,6 +10354,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -10910,7 +10927,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -10987,6 +11006,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -11175,7 +11195,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -11252,6 +11274,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -11584,7 +11607,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -11661,6 +11686,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -12385,7 +12411,9 @@ echo $ECHO_N "checking for sigprocmask..
 if test "${ac_cv_func_sigprocmask+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -12461,6 +12489,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { echo "$as_me:$LINENO: result: $ac_cv_func_sigprocmask" >&5
 echo "${ECHO_T}$ac_cv_func_sigprocmask" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test $ac_cv_func_sigprocmask = yes; then
   gl_cv_func_sigprocmask=1
 fi
@@ -14665,7 +14694,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -14742,6 +14773,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -15119,7 +15151,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -15196,6 +15230,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -15446,7 +15481,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -15523,6 +15560,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -15547,7 +15585,9 @@ echo $ECHO_N "checking for $ac_func... $
 if { as_var=$as_ac_var; eval "test \"\${$as_var+set}\" = set"; }; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  cat >conftest.$ac_ext <<_ACEOF
+  mirsave_CPPFLAGS=$CPPFLAGS
+CPPFLAGS="-D_MIRMAKE_H $CPPFLAGS"
+cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
 cat confdefs.h >>conftest.$ac_ext
@@ -15624,6 +15664,7 @@ fi
 ac_res=`eval echo '${'$as_ac_var'}'`
 	       { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
+CPPFLAGS=$mirsave_CPPFLAGS
 if test `eval echo '${'$as_ac_var'}'` = yes; then
   cat >>confdefs.h <<_ACEOF
 #define `echo "HAVE_$ac_func" | $as_tr_cpp` 1
@@ -16877,7 +16918,7 @@ exec 6>&1
 # values after options handling.
 ac_log="
 This file was extended by GNU M4 $as_me 1.4.9, which was
-generated by GNU Autoconf 2.61.  Invocation command line was
+generated by GNU Autoconf 2.61-MirPorts-1.  Invocation command line was
 
   CONFIG_FILES    = $CONFIG_FILES
   CONFIG_HEADERS  = $CONFIG_HEADERS
@@ -16930,7 +16971,7 @@ _ACEOF
 cat >>$CONFIG_STATUS <<_ACEOF
 ac_cs_version="\\
 GNU M4 config.status 1.4.9
-configured by $0, generated by GNU Autoconf 2.61,
+configured by $0, generated by GNU Autoconf 2.61-MirPorts-1,
   with options \\"`echo "$ac_configure_args" | sed 's/^ //; s/[\\""\`\$]/\\\\&/g'`\\"
 
 Copyright (C) 2006 Free Software Foundation, Inc.
